### PR TITLE
Bump version to 0.6.0

### PR DESCRIPTION
**Pull request by jverkoey**
Follow-up step after merging this pull request: `bundle exec rake release`

### CODE REVIEW

**Comment by jverkoey** [comment]
Ideas for names:

- Document all the things!

File: CHANGELOG.md [hunk 1]
```diff
@@ -140,7 +148,7 @@
 
 ##### Enhancements
 
-* Support for documenting Objective-C projects! 🎉
+* Support for documenting Objective-C projects! 
```
Comment (jpsim) [inline CHANGELOG.md]: oops, unintentional change?

**Comment by jpsim** [comment]
Looks like a number of the integration specs failed. Have you looked into those?

**Comment by jverkoey** [comment]
Haven't looked into them yet. Will try to this week :)

**Comment by esad** [comment]
Should we include https://github.com/realm/jazzy/pull/466 in the changelog -  some users may wonder why the behavior had changed?

**Comment by jpsim** [comment]
Added in a485292, thanks @esad!

**Comment by jpsim** [comment]
Integration specs have now been fixed / updated and I'm reviving this release in #515. I'll complete the release there if everything looks good.

Thanks all!

**Pull request closed without merging**